proto/xudrpc.proto

```diff
@@ -313,6 +320,17 @@ message ChannelBalanceResponse {
   map<string, ChannelBalance> balances = 1 [json_name = "orders"];
 }
 
+message OpenChannelRequest {
+  // The node pub key of the peer which to open channel with.
+  string node_pub_key = 1 [json_name = "peer_pub_key"];
+  // The ticker symbol of the currency to open channel for.
+  string currency = 2 [json_name = "currency"];
+  // The amount of the channel in the smallest base unit of the currency.
```

Comment:
I think we should denominate this in satoshis rather than the smallest base unit, otherwise we will quickly run into integer overflow issues with the raiden currencies using 18 digits after the decimal place as we've encountered with the other rpc calls. This would also require changes to the logic elsewhere.

Reply:
Thanks for noticing. Yep, in the context of RPC we're always denominating amounts in satoshis per coin. Will implement the required logic.